RealDeviceMap-UIControl/Misc.swift

```diff
@@ -299,8 +300,8 @@ extension XCTestCase {
 
     }
 
-    func isStartup(screenshot: XCUIScreenshot?=nil) -> Bool {
-
+    func isStartup(click: Int = 0, screenshot: XCUIScreenshot?=nil) -> Bool {
```

Comment:
shouldn't his be a `Bool`?

Reply:
It could be. 0/1 worked well for me. I didn't want to add another bool to the function as the function returns a bool value. For clarity of code, it made sense to me to not create the variable as a bool. It could easily be if necessary. It does not effect code execution at all whether or not it's bool or int.

Reply:
Yeah change it to Bool. Using an Int instead as Bool makes code unclear.